backend/src/main/java/gov/cdc/usds/simplereport/db/repository/DeviceSpecimenTypeRepository.java

```diff
@@ -24,5 +24,9 @@
   @EntityGraph(attributePaths = {"deviceType", "specimenType"})
   Optional<DeviceSpecimenType> findFirstByDeviceTypeInternalIdOrderByCreatedAt(UUID deviceTypeId);
 
+  @EntityGraph(attributePaths = {"deviceType", "specimenType"})
+  Optional<DeviceSpecimenType> findByDeviceTypeInternalIdAndSpecimenTypeInternalIdOrderByCreatedAt(
+      UUID deviceTypeId, UUID specimenTypeId);
```

Comment:
should we be using `BASE_QUERY` here to make sure the DST hasn't been soft deleted? it doesn't even look like we soft delete these instances ever so maybe not necessary unless we're planning on fixing that in `gov.cdc.usds.simplereport.service.DeviceTypeService#updateDeviceType`
I wish we didn't say that we [soft-delete them when we don't](https://github.com/CDCgov/prime-simplereport/blob/517aac1e2387351173ccff8ce84a2b3c4d1cc210/backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSpecimenType.java#L9-L12)

Reply:
We dont soft delete them, we delete them since its a joint table, that's the motivation behind this effort and why we're getting rid of the DeviceSpecimenType object.

Adding this check was done to maintain compatibility with backend code to limit changes to the frontend, the next PR will address the backend changes.

Reply:
oh right so this method won't even be necessary in the final version